modules/meilisearch/meilisearch.go

```diff
@@ -0,0 +1,132 @@
+package meilisearch
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultMasterKey = "just-a-master-key-for-test"
+	defaultHTTPPort  = "7700/tcp"
+)
+
+// MeilisearchContainer represents the Meilisearch container type used in the module
+type MeilisearchContainer struct {
+	testcontainers.Container
+	MasterKey string
+}
+
+// Deprecated: use Run instead
+// RunContainer creates an instance of the Meilisearch container type
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*MeilisearchContainer, error) {
+	return Run(ctx, "getmeili/meilisearch:v1.10.3", opts...)
+}
+
+// Run creates an instance of the Meilisearch container type
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*MeilisearchContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        img,
+		ExposedPorts: []string{defaultHTTPPort},
+		Env: map[string]string{
+			"MEILI_MASTER_KEY": defaultMasterKey,
```

Comment:
idea: I'd extract this string to a constant, as it's used three times in the module.